edb/common/markup/__init__.py

```diff
@@ -33,7 +33,7 @@ class MarkupCapableMeta(type):
     def __new__(mcls, name, bases, dct):
         cls = super().__new__(mcls, name, bases, dct)
         if 'as_markup' in dct:
-            serializer.serializer.register(cls)(getattr(cls, 'as_markup'))
```

Comment:
Explanation: getattr() with a string literal and no third argument is the same as a regular attribute get.